vignettes/approx-inference.Rmd

```diff
@@ -110,6 +110,15 @@ sdlog <- 0.5
 obs_time <- 25
 sample_size <- 200
 
+filter_obs_by_obs_time <- function(linelist, obs_time) {
```

Comment:
I really don't like putting function definitions into the vignettes like this. I think all we need is the filter line so can you just use that as needed vs this approach

Reply:
Sure I agree that is probably better. This is quicker to do now and could be quicker when we do the refactor to a future function